docs/en/changes/changes.md

```diff
@@ -8,6 +8,7 @@
 
 #### UI
 * General service instance: move `Thread Pool` from JVM to Overview, fix `JVM GC Count` calculation.
+* Add Apache ShenYu (incubating) component LOGO.
```

Comment:
Another one is missing.

Reply:
I've made it up. Please help me review it